slider/curve.py

```diff
@@ -39,18 +40,12 @@ def __call__(self, t):
         raise NotImplementedError('__call__')
 
     def __init_subclass__(cls):
-        cls._kind_dispatch
-        for meth in super(cls, cls).__abstractmethods__:
-            impl = getattr(cls, meth)
-            if impl.__doc__ is None:
-                impl.__doc__ = getattr(super(cls, cls), meth).__doc__
-
         for kind in cls.kinds:
             cls._kind_dispatch[kind] = cls
 
 
 class Bezier(Curve):
-    kinds = 'L'
+    kinds = ()
```

Comment:
yeah, I guess we should just have a map internally there, no need for this registration magic